File: Pattern 8/Pattern_8.java
class Solution {

    void printTriangle(int n) {
        for(int i =n-1; i>=0; i--) {
	        for(int j =n-1; j>i; j--) {
	            System.out.print(" ");
	        }
	        for(int j =2*i+1; j>0; j--) {
	            System.out.print("*");
	        }
	        System.out.println();
	    }
    }
}

File: Pattern 8/Pattern_8.py
class Solution:
    def printTriangle(self, n):
        for i in range(n-1,-1,-1):
            for j in range(n-1,i,-1):
                print(end = " ")
            for j in range(2*i+1,0,-1):
                print(end = "*")
            print()
	        
